packages/user/src/model/users/controller.ts

```diff
@@ -48,6 +49,21 @@ const plugin = async (
     }
   );
 
+  fastify.get(
+    ROUTE_ME,
+    {
+      preHandler: fastify.verifySession(),
+    },
+    async (request: SessionRequest, reply: FastifyReply) => {
+      const service = new Service(request.config, request.slonik);
+      const userId = request.session?.getUserId();
+
+      if (userId) {
```

Comment:
a route should always return a reply. what if in any case userId is null/undefined? what will be the response?
I think in that case we need to log the error and throw 500 http error to user.

Reply:
Since this route completely depends upon the session,
and execute only with valid session,
there will always be the userId during its execution.

Reply:
we cannot assume that always, there might be unexpected cases where there might be null/undefined userId from session in that case this request will be stock, so in any circumstances we need to send reply.